tensorboard/webapp/metrics/store/metrics_types.ts

```diff
@@ -37,10 +37,7 @@ import {
   TooltipSort,
   XAxisType,
 } from '../types';
-import {
-  ColumnHeader,
-  DataTableMode,
-} from '../views/card_renderer/scalar_card_types';
+import {ColumnHeader, DataTableMode} from '../../widgets/data_table/types';
```

Comment:
Did your linter settings get changed somehow? When I change this the autoformatter reverts the change.

Reply:
I think it is dependent on the trailing comma or not?

Reply:
It seems like it has to do with the length of the import. When I tried earlier I hadn't patched your full PR, but having done so now there are no formatting issues so this makes sense.